modules/hcp-ibmz-update-reg-ca.adoc

```diff
@@ -0,0 +1,65 @@
+:_mod-docs-content-type: CONCEPT
+[id="hcp-ibmz-update-registry-ca-private-registry_{context}"]
```

Comment:
```suggestion
[id="hcp-ibmz-update-registry-ca_{context}"]
```

Reply:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections:~:text=The%20anchor%20ID%20with%20_%7Bcontext%7D%20must%20be%20lowercase%20and%20match%20the%20module%E2%80%99s%20file%20name.